### ISSUE TITLE
Protect Proposals and Sketches from Edits by Others.

### ISSUE THREAD

**Issue by parabolika66** [MyrionPhoenixmoon/scenarioo#15]
As a user I don't want others to edit my proposals and sketches so that my ideas don't vanish.

**Comment by adiherzog** [comment]
This requires a proper user management.

**Comment by MyrionPhoenixmoon** [comment]
That is true. But a tiny part of it can and should be implemented already in the MVP:
While a proposal is being edited / a step is being sketched, it must be locked to others. 
A basic form of mutex should suffice.
